CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+3.1.2 (October 2017)
+--------------------
+
+- Fix display of channel buttons in IE11
+- Add IDR figure to demo
+- Remove extra computation when converting radians and degrees. Thanks to https://github.com/carandraug
```

Comment:
Very minor nitpick but I'd put the credit in brackets at the end rather than starting a new sentence:
"... degree (thanks to https://github.com/carandraug)"